clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -117,6 +117,10 @@ Changes in existing checks
   options `HeaderFileExtensions` and `ImplementationFileExtensions` by the
   global options of the same name.
 
+- Improved :doc:`bugprone-too-small-loop-variable
+  <clang-tidy/checks/bugprone/too-small-loop-variable>` check by correctly 
+  implementing the check for const loop variable initialized with a function declaration.
```

Comment:
change this second 'check' to 'support' to avoid duplicates, and its not const loop variable but an const loop boundary, and you can remove that "initialized with a function declaration", because this will work also for global variables.